joomla/joomla-cms
# [4.0] fix webp support

#31048 added support for webp images however it was not complete. MediaHelper::isImage was not updated to support webp. As a result while webp could be added to media manager as an allowed image format everything worked except for the display of the thumbnail.

As this PR fixes that we might as well add webp to the default list of supported image formats.

### To test

1. either do a fresh install with the prebuilt package OR apply this pr, run npm ci and go to the media manager options. delete all the values in the Legal Extensions (File Types), Legal Image Extensions (File Types) and Legal MIME Types and then save the options. This will reset the value to the new defaults.
2. Try to upload any webp image
3. Try to crop and resize the image
4. Try to rename the image
5. Get a shareable link and make sure it works

### Before
![image](https://user-images.githubusercontent.com/1296369/114272607-be6d8400-9a0e-11eb-9217-ef2dbe8cfbe3.png)

### After
![image](https://user-images.githubusercontent.com/1296369/114272628-cdeccd00-9a0e-11eb-8c91-875207a7b7df.png)
